### ISSUE TITLE
Learning rate on pretrained model

### ISSUE THREAD

**Issue by snownus** [loshchil/SGDR#1]
if the model is pretrained in a good state, and I want to apply your models. How do I set the reasonable learning rate? If the initial learning rate is too large, then it may go out too far of the optimized region.

**Comment by loshchil** [comment]
it depends on what you want to achieve and how many epochs you are able to invest. 
You may try to run cosine annealing with some relatively small learning rate and for some relatively small number of epochs. Then, if you don't diverge, you can try to double the initial learning rare and increase the total number of epochs. There might be other ways to do that.

**Comment by snownus** [comment]
Thanks. 

I found that Tensorflow has implemented your methods. 
https://www.tensorflow.org/api_docs/python/tf/train/cosine_decay_restarts

**Comment by loshchil** [comment]
Yes, it was implemented by @PatrykChrabaszcz
